src/app/portlets/dot-starter/dot-starter.component.html

```diff
@@ -205,7 +220,12 @@ <h4>{{ 'starter.footer.link.examples.title' | dm }}</h4>
         </a>
     </div>
     <div class="p-lg-4 p-md-12">
-        <a href="#" data-testid="starter.footer.link.community" class="dot-starter-bottom__block">
+        <a
+            href="https://dotcms.com/forum/"
+            data-testid="starter.footer.link.community"
```

Comment:
Isn't should this be camelCase? we need to doc this thing we keep having this issues.